boot/pod/src/boot/file.clj

```diff
@@ -21,7 +21,7 @@
 (def ^:dynamic *sync-delete* true)
 (def ^:dynamic *hard-link*   true)
 
-(def windows? (boot.App/isWindows))
+(def ^:dynamic windows? (boot.App/isWindows))
```

Comment:
I mean it is just a minor thing anyways, given that we don't know all the implications here we might need to wait better opinion.